docs/index.rst

```diff
@@ -157,6 +157,19 @@ value``) pairs:
   [(<function receive_data at 0x...>, 'received!')]
 
 
+Muting signals
+--------------
+
+To mute a signal, as may be required when testing, the
+:meth:`~Signal.mute` can be used as a context decorator:
```

Comment:
I think you meant `muted`

Reply:
On the other hand, `mute()` seems like a better choice for the method name. I will leave it up to you to choose one version.